unittest/CMakeLists.txt

```diff
@@ -82,6 +82,15 @@ IF(LUA5_1_FOUND)
   ADD_TEST_CFLAGS(lua '-DPINOCCHIO_SOURCE_DIR=\\\"${${PROJECT_NAME}_SOURCE_DIR}\\\"')
 ENDIF(LUA5_1_FOUND)
 
+IF(${BUILD_PYTHON_INTERFACE} STREQUAL "ON")
+  IF(EIGENPY_FOUND)
+    FINDPYTHON(2.7 EXACT REQUIRED)
+    ADD_UNIT_TEST(python "eigen3;eigenpy")
```

Comment:
I think python is not a good choice for the name to avoid conflict with the python interpreter. python_parser may be better.